### DISCUSSION TITLE
RDS instance in tenant - not autoscaling

### DISCUSSION THREAD

**Discussion by tasneemfazl** [awslabs/aws-saas-boost#102] [Q&A] [answered]
Dear All, 
The RDS instance spinned through SaaSBoost has autoscaling disabled and we are unable to change the instance type from magnetic to general or IOPS. A little bit probing has given details that SaaSBoost is spinning RDS instance with Magentic storage hence we are unable to change the storage, autoscale or change DB instance storage from magnetic to others.

can anyone here support me how to fix this? I want to have RDS instance type changed.

https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#CHAP_Storage.Magnetic

The following are some limitations for magnetic storage:
Doesn’t allow you to scale storage when using the SQL Server database engine.
Doesn’t support storage autoscaling.
Doesn’t support elastic volumes.
Limited to a maximum size of 3 TiB.
Limited to a maximum of 1,000 IOPS.

**Comment by brtrvn** [comment]
@tasneemfazl can you please enter a Feature Request issue to change CloudFormation's default of magnetic storage to RDS's suggested default of SSD (general purpose) storage?

**Reply by tasneemfazl** [reply]
@brtrvn how do I raise the request ?

**Reply by brtrvn** [reply]
Simply go to the [Issues](https://github.com/awslabs/aws-saas-boost/issues) tab and click on the **New issue** button and then the **Get started** button for Feature Request.

Easy to modify SaaS Boost to default to general purpose SSD vs. magnetic storage.

**Comment by brtrvn** [comment, marked as answer]
We changed the default storage type to general purpose SSD with PR #104
